chart/templates/deployment.yaml

```diff
@@ -104,7 +104,7 @@ spec:
             - name: PUBSUB_URL
               value: 'nats://keptn-nats-cluster'
             - name: PUBSUB_TOPIC
-              value: 'sh.keptn.event.monitoring.configure,sh.keptn.event.get-sli.triggered'
+              value: 'sh.keptn.event.monitoring.configure,sh.keptn.event.configure-monitoring.triggered,sh.keptn.event.get-sli.triggered'
```

Comment:
In the future we will also need to listen to `sh.keptn.event.configure-monitoring.triggered` - this is a pending change though (https://github.com/keptn/keptn/issues/6805 ).